CriteoAdViewer/Sources/AdViewer/AdViewerViewController.swift

```diff
@@ -177,6 +206,17 @@ class AdViewerViewController: FormViewController {
     }
   }
 
+  private func updateCoppa() {
+    switch coppa {
+    case .active:
+      criteo?.childDirectedTreatment = NSNumber(booleanLiteral: true)
```

Comment:
isn't it possible to use a true/false constant ?

Reply:
The `childDirectedTreatment` is a 3-state property. A boolean cannot map the possible values.